argoverse/utils/sim2.py

```diff
@@ -77,7 +95,7 @@ def matrix(self) -> np.ndarray:
 
     def compose(self, S: "Sim2") -> "Sim2":
         """Composition with another Sim2."""
-        return Sim2(self.R_ * S.R_, ((1.0 / S.s_) * self.t_) + self.R_ @ S.t_, self.s_ * S.s_)
+        return Sim2(self.R_ @ S.R_, ((1.0 / S.s_) * self.t_) + self.R_ @ S.t_, self.s_ * S.s_)
```

Comment:
Could we consider splitting this up into a few lines? It's tough for me to read all of the operations on one line.

Reply:
+1